modules/math/src/geometry/primitives/modes.ts

```diff
@@ -33,9 +35,13 @@ export function isPrimitiveModeExpandable(mode) {
       return false;
   }
 }
-
-/** @type {types['getPrimitiveModeExpandedLength']} */
-export function getPrimitiveModeExpandedLength(mode, length) {
+/**
+ * Returns new length depends on glType
+ * @param mode
+ * @param length
+ * @returns new length
+ */
+export function getPrimitiveModeExpandedLength(mode: number, length: number): number | void {
```

Comment:
Same here, throw exception instead of `assert` so that you can guarantee that this function always returns a `number` (it either returns a number or throws an error)